zfs/replicate/filesystem/create.py

```diff
@@ -35,6 +35,8 @@ def create(filesystem: FileSystem, ssh_command: str) -> None:
         if proc.returncode:
             if b"successfully created, but not mounted" in error:
                 return  # Ignore this error.
+            if b"filesystem successfully created, but it may only be mounted by root" in error:
```

Comment:
I think we should merge this into checking if any of the ignored errors are substrings of the error to avoid so many conditionals.  Do you mind making that change before we merge this?